src/pages/Thanks/ThanksPage.vue

```diff
@@ -546,9 +546,10 @@ export default {
 		this.enableMayChallengeHeader = shareChallengeExpData?.version === 'b';
 
 		this.optedIn = data?.my?.communicationSettings?.lenderNews || this.$route.query?.optedIn === 'true';
+		const usedKivaCard = Number(this.receipt?.totals?.kivaCardTotal);
```

Comment:
Does this cover only the situation Shehaam mentioned? I thought it was for print cards. Maybe we can't call out the specific situation that she mentioned in Slack.